task/01-strings-tasks.js

```diff
@@ -268,8 +271,10 @@ function isString(value) {
  *   'Q♠' => 50
  *   'K♠' => 51
  */
-function getCardId(value) {
-  throw new Error('Not implemented');
+function getCardId(value) {  
+  const suits = ['♣', '♦', '♥', '♠'];
+  const values = ['A', '2', '3', '4', '5', '6', '7', '8', '9', '10', 'J', 'Q', 'K'];
+  return [].concat.apply([], suits.map(suit => values.map(v => v + suit))).indexOf(value);
```

Comment:
In this case, it's not necessary to recreate the array of all cards. Instead, you can use indexes of value and suite as a coefficient and calculate position:
e.g: Q♥ = 11 + (13 * 2) = 35, where 11 - position of value in the values array, 13 - length of values array and 2 - index of a suit